src/main/java/mate/academy/spring/model/CinemaHall.java

```diff
@@ -5,45 +5,17 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Entity
 @Table(name = "cinema_halls")
+@Data
+@NoArgsConstructor
```

Comment:
<img width="669" alt="image" src="https://user-images.githubusercontent.com/95774539/207011997-4f8c034b-1556-4f46-b8c9-ad5deaaa011b.png">
lombook annotation Data create only constructor with arguments, so that why I adding @NoArgsConstructor